CMSgov/design-system
# [WNMGDS-829] Fix docs site nav url

## Summary
- Fix navigation url for document site pages on mobile showing `access denied error`
- Fix USA banner overflowing the content, resulting in horizontal scrolling on mobile

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-829/fix-docs-site-nav-url/
1. Go `http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-829/fix-docs-site-nav-url/` on mobile phone
2. Click on `Menu`, then `Components`
3. Select 'Alert' 
4. Check that the page will load (check other nav links also)
5. Check also that the USA banner at top of the mobile screen fits
